site/src/@ndlib/gatsby-theme-marble/components/Shared/SearchTools/SearchFilterBox/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import {
+  SearchBox,
+  SelectedFilters,
+  ResetFilters,
+} from 'searchkit'
+import HeroBox from '@ndlib/gatsby-theme-marble/src/components/Shared/HeroBox'
+
+import banner from 'assets/images/banner.swirl.png'
+
+const SearchFilterBox = () => {
+  const fields = ['name', 'creator', 'allMetadata']
+  return (
+    <HeroBox backgroundImage={banner}>
+      <SearchBox
+        queryFields={fields}
+        queryBuilder={customQueryBuilder}
+      />
+      <SelectedFilters />
+      <ResetFilters />
+    </HeroBox>
+  )
+}
+
+const customQueryBuilder = (query, options) => {
```

Comment:
Let's try removing the customQueryBuilder here and instead import it from the original module. 
Change 
```
const customQueryBuilder = (query, options) => {
```
to 
```
export const customQueryBuilder = (query, options) => {
``` 
in the original then in the shadow component here:
```
import { customQueryBuilder } from '@ndlib/gatsby-theme-marble/src/components/Shared/SearchTools/SearchFilterBox
```